tutorials/not_so_obvious_python_stuff.ipynb

```diff
@@ -1693,21 +1648,19 @@
    "metadata": {},
    "source": [
     "Who has not stumbled across this quote \"we are all consenting adults here\" in the Python community, yet? Unlike in other languages like C++ (sorry, there are many more, but that's one I am most familiar with), we can't really protect class methods from being used outside the class (i.e., by the API user).  \n",
-    "All we can do is to indicate methods as private to make clear that they are better not used outside the class, but it is really up to the class user, since \"we are all consenting adults here\"!  \n",
+    "All we can do is indicate methods as private to make clear that they are not to be used outside the class, but it really is up to the class user, since \"we are all consenting adults here\"!  \n",
     "So, when we want to mark a class method as private, we can put a single underscore in front of it.  \n",
     "If we additionally want to avoid name clashes with other classes that might use the same method names, we can prefix the name with a double-underscore to invoke the name mangling.\n",
     "\n",
-    "This doesn't prevent the class user to access this class member though, but he has to know the trick and also knows that it his own risk...\n",
+    "This doesn't prevent the class user to access this class member though, but they have to know the trick and also know that it is at their own risk...\n",
```

Comment:
Thanks for the fix, could you please also change "the class user" to "the class users" to fix the grammar?